week-2/L-array-methods-2/exercise.js

```diff
@@ -15,8 +15,8 @@ var everyone = [
   "Swathi"
 ];
 
-var firstFive; // complete this statement
-var lastFive; // complete this statement
+var firstFive = everyone.slice(0, 5); // complete this statement
+var lastFive = everyone.slice(2, 6); // complete this statement
```

Comment:
This works because we know the lengths. Can you think how to write it so that it works with an array of any length?